content/docs/user-guide/interactivity/physics/nvidia-physx/wind-provider.md

```diff
@@ -1,51 +1,71 @@
 
-1. Add a **PhysX Force Region** component to the entity. The **PhysX Force Region** component creates the wind force.
+   The PhysX collider box displays cones representing the wind force direction.
 
-1. In the **PhysX Force Region** component, specify **10.0** in the **Direction** property's **Y** component to set a direction for the wind force.
+    {{< image-width "/images/user-guide/interactivity/physics/nvidia-physx/physx-wind-entity.png" "800" "PhysX Wind entity setup" >}}
 
-1. In the **PhysX Force Region** component, specify **50.0** in the **Magnitude** property to create magnitude for the wind force.
+1. To test the wind provider, add an entity with an NVIDIA Cloth mesh. There is an example cloth prefab included in the NVIDIA Cloth Gem that you can drag into the viewport from **Asset Browser**. Navigate to `Gems\NvCloth\Assets\prefabs\Cloth` in Asset Browser. Locate the `cloth_locked_corners_two.prefab` and drag and drop the prefab into the viewport.
 
-   The PhysX collider box in **Perspective** displays cones and vectors representative of the wind force direction.
+1. Use the **Move** tool to place the cloth entity. If you are using the Local wind tag, you must place the cloth asset inside the PhysX Collider volume of the wind provider entity.
 
-   ![PhysX Wind entity setup](/images/user-guide/physx/physx/ui-physx-wind-entity.png)
+    {{< image-width "/images/user-guide/interactivity/physics/nvidia-physx/physx-wind-cloth-entity.png" "500" "PhysX wind cloth entity setup" >}}
 
-1. To test the wind provider, add an entity with a **Cloth** mesh. There is an example slice asset you can drag into **Perspective** from **Asset Browser**. Navigate to `Gems\NvCloth\Assets\slices` in **Asset Browser**. Locate the `cloth_locked_corners_two.slice` and drag and drop the slice into **Perspective**.
+1. The `cloth_locked_corners_two.prefab` has the local wind property enabled on it's **Cloth** component. This generates a local wind force from the Cloth component that overrides the wind force from the wind provider you created. The following steps disable the local wind force of the prefab so that you can view the results of the wind provider you created:
+        
+    1. In **Entity Outliner**, double click the {{< icon "prefab.svg" >}} **cloth_locked_corners_two** prefab to edit it.
 
-1. Use the **Move** tool to place the cloth entity. If you are using the **Local wind tag**, you must place the cloth asset inside the **PhysX Collider** volume.
+    2. Click the {{< icon "entity.svg" >}} **cloth_locked_corners_two** child entity to select it.
 
-    ![PhysX Wind entity setup](/images/user-guide/physx/physx/ui-physx-wind-cloth-entity.png)
+    3. With the {{< icon "entity.svg" >}} **cloth_locked_corners_two** entity selected, in **Entity Inspector**, in the Cloth component, disable the **Enable local wind** toggle.
```

Comment:
```
1. The `cloth_locked_corners_two.prefab` entity has the **Local wind tag** property enabled on its Cloth component. This generates a local wind force from the Cloth component that overrides the wind force from the wind provider that you created.

To turn off the local wind force of the prefab so that you can view the results of the wind provider, do the following:
    * In **Entity Outliner**, double-click the {{< icon "prefab.svg" >}} **cloth_locked_corners_two** prefab to edit it.
    * Click the {{< icon "entity.svg" >}} **cloth_locked_corners_two** child entity to select it.
    * With the child entity selected, in **Entity Inspector**, in the Cloth component, turn off **Enable local wind**.
```
* Reformatted step 13 and its substeps using our preferred [nested list formatting](https://docs.o3de.org/docs/contributing/to-docs/style-guide/format/#nested-lists).
* Spelling fix ("its")
* Dropped the bold formatting from "**Cloth**" (second reference to the Cloth component).
* Changed occurrences of "disable" to "turn off" (more inclusive phrasing).
* [Hyphenate "double click"](https://www.merriam-webster.com/dictionary/double-click)
* Shortened the third substep by using "child entity" instead of repeating the file name+icon.